broker-daemon/config.json

```diff
@@ -1,14 +1,42 @@
 {
+  "broker": {
```

Comment:
do these need to be grouped under `broker`? or can they be top level?

Reply:
@treygriffith they can be top level